hack/jenkins/test-image/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # This file creates a build environment for building and running kubernetes
 # unit and integration tests
 
-FROM golang:1.5.3
+FROM golang:1.6
```

Comment:
I think @luxas is right - we should be using `golang:1.6.0` here. Otherwise the Go version may or may not mysteriously jump when we push a new `kubekins-test` image.

As noted, we ran into issues depending on the `golang:1.4` image in the crossbuild image, since we weren't forcing a pull from upstream. On our Jenkins this got stuck on an old 1.4.2 image, until it happened to get re-pulled one day.

It's much better to be explicit so we know when we're jumping versions.